tests/sensu-client-windows.pp

```diff
@@ -0,0 +1,9 @@
+# Note: Check http://repositories.sensuapp.org/msi/ for the latest version.
+class { '::sensu':
+  version           => '0.26.5-1',
```

Comment:
Thanks for the catch, I noticed newer releases are in the 2012r2 directory.  Updating it to https://repositories.sensuapp.org/msi/2012r2/sensu-0.29.0-11-x64.msi